app/models/mdm/payload.rb

```diff
@@ -92,10 +92,18 @@ class Mdm::Payload < ApplicationRecord
 
   #
   # Serializations
-  #
 
-  serialize :urls
-  serialize :build_opts
+  if ActiveRecord::VERSION::MAJOR >= 7 && ActiveRecord::VERSION::MINOR >= 1
+    serialize :urls, coder: YAML
+  else
+    serialize :urls
+  end
```

Comment:
since specifying `coder:` is a noop, and the old code didn't specify _any_ coder, this `if/else` statementi s kind of redundant, I did add in an explicit `serialize :urls, YAML` but that broke the tests and it makes more sense to keep the old functionality the same as it was before
I don't mind keeping it here, since it makes it obvious what the intention is